index.html

```diff
@@ -3859,14 +3910,14 @@ <h2><a name="srgb-standard-colour-space"><span class="chunk">sRGB</span>
 
 <p>When the <span class="chunk">sRGB</span> chunk is present, it
 is recommended that decoders that recognize it and are capable of
-colour management <a href="#ICC"><span class=
-"bibref">[ICC]</span></a> ignore the <a href="#gama-image-gamma"><span
+colour management
+ignore the <a href="#gama-image-gamma"><span
```

Comment:
Just confirming this removal was intentional.
Maybe it was removed because the ISO site is down?

Reply:
No, it is because that is a rather spurious link for "colour management exists". 
It is to the bibliography, and just links to color.org - it is separate from the normative [[icc]] reference which is to ICC Profile 4.3.

Reply:
Okay. Makes sense to me. (It was rather odd.)
So these are indeed intentional, right?

Reply:
Yes.